docs/components/workflow.rst

```diff
@@ -36,12 +36,12 @@ DKAN Workflow is included on all out-of-the-box DKAN sites; however, it is not e
 
    drush en dkan_workflow -y
 
-Enabling DKAN workflow will automatically enable all other required modules and add the Workflow Supervisor role to all users already assigned the site manager role. (More information available in the `Workflow Roles`_
+Enabling DKAN workflow will automatically enable all other required modules and add the Workflow Supervisor role to all users already assigned the site manager role. (More information available in the :ref:`Workflow Roles`
```

Comment:
This reference is not working properly, see https://docs.getdkan.com/en/2166-rm-doc-images/components/workflow.html it directs the user to the same place.

Reply:
Thanks for catching that @dharizza, adding one last commit...I hope